scripts/zones/Meriphataud_Mountains/mobs/Coo_Keja_the_Unseen.lua

```diff
@@ -2,17 +2,55 @@
 -- Area: Meriphataud Mountains (119)
 --   NM: Coo Keja the Unseen
 -----------------------------------
+local ID = require("scripts/zones/Meriphataud_Mountains/IDs")
 mixins = {require("scripts/mixins/job_special")}
 -----------------------------------
 local entity = {}
 
+-- TODO: Implement better pathing systems for guards to follow master
+
 entity.onMobSpawn = function(mob)
     xi.mix.jobSpecial.config(mob, {
         specials =
         {
             {id = xi.jsa.MIJIN_GAKURE, hpp = math.random(10, 15)},
         },
     })
+
+    -- Takes half damage from all attacks
+    mob:addMod(xi.mod.UDMGPHYS,-5000)
+    mob:addMod(xi.mod.UDMGRANGE,-5000)
+    mob:addMod(xi.mod.UDMGMAGIC,-5000)
+    mob:addMod(xi.mod.UDMGBREATH,-5000)
+
+    -- May spawn in a party with two other Orcs
+    if math.random(1,2) == 1 then
```

Comment:
`if math.random(3) == 2 then` like you agreed to with the other ones of this type :+1: